src/prettytable/prettytable.py

```diff
@@ -2031,7 +2031,7 @@ def get_json_string(self, **kwargs) -> str:
         """
 
         options = self._get_options(kwargs)
-        json_options = dict(indent=4, separators=(",", ": "), sort_keys=True)
+        json_options: Any = dict(indent=4, separators=(",", ": "), sort_keys=True)
```

Comment:
At least a dict:

```suggestion
        json_options: dict = dict(indent=4, separators=(",", ": "), sort_keys=True)
```

Ideally we should list the types of the keys and values, but that can wait for `--strict`.